### ISSUE TITLE
why does this depend on sys/ioctl.h

### ISSUE THREAD

**Issue by veso266** [beandog/dvd_info#3]
Hi there, windows user here that would like to use this :smile: 

well here we go: I tried to compile this on Mingw like this
```bash
install libdvdread
	pacman -S mingw-w64-i686-libdvdread
	pacman -S mingw-w64-x86_64-libdvdread

autoreconf -vif
autoreconf
automake
./configure
make
make install
```

and it started to complain sys/ioctl.h which windows doesn't have so I cannot compile that :cry: 

the onlything I would like to do is get chapters from dvd (I have DVD in iso file) and when I open it in VLC I can in this dvd menu select a specific chapter (but its not a chapter, this is one big chapter and in dvd menu I can get to a specific point in this chapter, not sure how this is called, would like to extraxt this big chapter into this smaller ones that are on DVD menu :smile: )

file in question is here: https://mega.nz/file/rokFGJbZ#PhCm_J3-tBTfT_7_HSB0QIwwunyWF0y_Y4iD5_L1B2o

would just like to split this movie in specific parts :smile: 

Thanks for Anwsering and Best Regards

**Comment by beandog** [comment]
I can't duplicate the bug, but that doesn't mean it's not broken.

ioctl is needed to check DVD drive status. However, it's just Linux and *BSD systems that should be running it, and the include should only be on those drive tool headers.

**Comment by veso266** [comment]
Hi there, thanks for talking your time to respond, I dont think this is a bug, its just that I am trying to use your tool for something its not indended to (running it in windows 😄 )
I only needed dvd_info to extraxt chapters from ripped dvd in iso format and at the end I found a diferent tool for this job: https://github.com/thierer/lsdvd

So as for me, you can close this issue

**Comment by beandog** [comment]
I got it fixed anyway, it'll be part of the 1.12 release. If you don't mind testing it, that'd be good.

**Comment by veso266** [comment]
ugh finally fixed it
![slika](https://user-images.githubusercontent.com/6347325/93072056-2af2f100-f681-11ea-87eb-333898cec2e3.png)
I had to fix some windows specifics like mkdir and open but at the end it works as shown in the image, if you want you can merge my pull request: https://github.com/beandog/dvd_info/pull/4

Thanks for talking time to anwser and investigate :smile:
